docs/components/page-higher-order/withDocumentationElements.tsx

```diff
@@ -21,7 +21,9 @@ const withDocumentationElements = (meta: PageMetadata) => {
           asPath={router.asPath}
           sourceCodeUrl={meta.sourceCodeUrl}
           tocVisible={!meta.hideTOC}>
-          <MDXProvider components={components}>{props.children}</MDXProvider>
+          <MDXProvider components={(components as unknown) as Components}>
```

Comment:
This one is fishy 🤔 
Is `import * as components from '~/common/translate-markdown';` breaking typings 🤔 ?

Reply:
Indeed, even if `translate-markdown` exports correct typing, TS has problems with imports and this was the only simple way.